app/views/layouts/application.html.haml

```diff
@@ -1,9 +1,22 @@
 !!! 5
 %html
   %head
-    %title Qna
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```

Comment:
Personally, I would prefer this syntax for attributes
```haml
%meta(content='text/html' charset='utf-8 http-equiv='Content-Type')
```
because it's shorter, even though haml supports both. I want to set up the haml linter sometime, but I haven't gotten to it. Also, what is the trailing `/` for?